Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/Persona.java

```diff
@@ -376,7 +385,8 @@ private static Persona getPersonaByUUID(String uuid) throws CentralRepoException
     public static Collection<Persona> getPersonaByName(String partialName) throws CentralRepoException {
 
         String queryClause = PERSONA_QUERY
-                + "WHERE LOWER(p.name) LIKE " + "LOWER('%" + partialName + "%')" ;
+                + "WHERE p.status_id != " + PersonaStatus.DELETED.status_id + 
```

Comment:
I think other methods using the PERSONA_QUERY also need to filter out deleted personas.  
We should probably just add the WHERE condition to PERSONA_QUERY. 

Similar check probably needs to be added to queries in PersonaAccount ?